lib/Doctrine/DBAL/Platforms/SqlitePlatform.php

```diff
@@ -535,9 +535,12 @@ public function getName()
     public function getTruncateTableSQL($tableName, $cascade = false)
     {
         $tableIdentifier = new Identifier($tableName);
-        $tableName       = str_replace('.', '__', $tableIdentifier->getQuotedName($this));
+        $quotedTableName = str_replace('.', '__', $tableIdentifier->getQuotedName($this));
 
-        return 'DELETE FROM ' . $tableName;
+        $deleteSql        = 'DELETE FROM ' . $quotedTableName;
+        $resetSequenceSql = 'DELETE FROM sqlite_sequence WHERE name = ' . $this->quoteStringLiteral($tableName);
+
+        return $deleteSql . '; ' . $resetSequenceSql;
```

Comment:
Usually, DB drivers for PHP do not allow multiple statements in a call for security reasons. Is SQLite an exception or all PDO-based drivers work like this?